MineSharp.Bot/Plugins/ChatPlugin.cs

```diff
@@ -581,7 +581,7 @@ private Task HandleChatInternal(Uuid? sender, ChatComponent.Chat message, ChatMe
         ChatComponent.Chat? content,
         ChatComponent.Chat? target)
     {
-        var entry = Bot.Registry["minecraft:chat_type"]["value"][index];
```

Comment:
The vanilla server 1.20.1 and 1.20.4 is sending the registries with the minecraft: namespace prefix.
A proper implementation of the registry including identifiers is probably necessary for the future...

Reply:
I had to remove the minecraft prefix otherwise I got an exception whenever the bot received a chat message

Reply:
I resolve this as I think this is fixed